homeassistant/components/tradfri/sensor.py

```diff
@@ -1,10 +1,9 @@
 """Support for IKEA Tradfri sensors."""
 import logging
 
-from homeassistant.components.tradfri.base_class import TradfriBaseDevice
+from .base_class import TradfriBaseDevice
```

Comment:
Please sort the package imports last. If you use isort it will do that automatically I think.

Reply:
I ran optimize imports in pycharm, which seems to do the same